packages/drivers/routerlicious-driver/src/gitManager.ts

```diff
@@ -0,0 +1,316 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
```

Comment:
I feel like we should either open a bug to dedup the code, or make a call that we will break depending from server here (and maybe add a comment here that it originally was cloned form another places).
I prefer to duplicate forever and remove any dependency on server, but I have not checked where else we depend on server in R11s code base.

@anthony-murphy , @tylerbutler  - FYI, this is cloning server code to break dependency and improve substantially developer velocity.

Reply:
I will open a bug to break the dependency entirely.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/4077 - Work item